source/renderer/app/components/staking/stake-pools/StakePoolsTable.scss

```diff
@@ -99,14 +99,18 @@
     thead {
       background: var(--theme-bordered-box-background-color);
       border-radius: 4px 4px 0 0;
-      box-shadow: 0 10px 10px -10px rgba(0, 0, 0, 0.2);
       display: block;
       margin: -20px -20px 0;
       padding: 0 20px;
       position: sticky;
       top: 0;
+      transition: box-shadow 0.1s ease-in-out;
       z-index: 999;
 
+      &.headerStuck {
```

Comment:
I'm not a native speaker, but this name sounds a bit unnatural to me, perhaps I would call it `fixed` rather than `stuck`. Maybe @shawnbusuttil or @ManusMcCole will be able to advise?

Reply:
@marcin-mazurek thats is in reference to `position: sticky;`
Do you think `headerSticky` sounds better?

Reply:
I think your intention was to use past tense for sticky? The dictionary suggests **stickied**, but it sounds equally strange. That's why I suggested `fixed`, but perhaps that would be confusing too (someone may think that that `position: fixed` is used under the hood). `.headerSticky` works!

Reply:
Yes, headerSticky sounds better. Thanks @marcin-mazurek @shawnbusuttil